BUILD

```diff
@@ -1810,12 +1810,12 @@ grpc_cc_library(
     name = "grpc_mock_cel",
     hdrs = [
         "src/core/lib/security/authorization/mock_cel/activation.h",
-        "src/core/lib/security/authorization/mock_cel/cel_expr_builder_factory.h",
-        "src/core/lib/security/authorization/mock_cel/cel_expression.h",
+        #"src/core/lib/security/authorization/mock_cel/cel_expr_builder_factory.h",
```

Comment:
I assume these lines are being commented out just for testing.

Reply:
This is actually intended because those headers are not able to be compiled so it should be excluded from the build file. Previously those headers are not included by sources, it happened to pass the build tests.

Reply:
@ashithasantosh If these files can't actually be compiled, we should fix that.

Reply:
Sure! Will look into it after clang-tidy update is merged.